### ISSUE TITLE
Cannot run "deno compile" on Fresh project with Tailwind support (node_modules issue)

### ISSUE THREAD

**Issue by cknight** [denoland/deno#21476]
Version: Deno 1.38.5

When attempting to compile a Fresh project which has the new Tailwind support enabled it fails as follows:
```ts
chris@chris-desktop:~/dev/deno/compileTest/fresh-project$ deno compile -A main.ts
Check file:///home/chris/dev/deno/compileTest/fresh-project/main.ts
Compile file:///home/chris/dev/deno/compileTest/fresh-project/main.ts to fresh-project
error: Writing fresh-project

Caused by:
    0: Reading symlink /home/chris/dev/deno/compileTest/fresh-project/node_modules/.deno/chokidar@3.5.3/node_modules/fsevents
    1: No such file or directory (os error 2)
```

Steps to reproduce:
1. Init a new Fresh project (`deno run -A -r https://fresh.deno.dev`)
2. When prompted, choose 'y' for Tailwind support and 'y' for use of VSCode
3. run `deno compile -A main.ts`

**Comment by dsherret** [comment]
Does that path exist? Are you using `"nodeModulesDir": true`? Maybe try deleting the `node_modules` directory and trying again.

**Comment by cknight** [comment]
Simpler reproduction.

deno.json
```json
{
  "lock": false,
  "imports": {
    "tailwindcss": "npm:tailwindcss@3.3.5"
  },
  "nodeModulesDir": true
}
```

main.ts
```ts
import { type Config } from "tailwindcss";

console.log("hello world");
```

run `deno compile main.ts`

**Comment by dsherret** [comment]
It's because of the optional dependency. I'm looking into fixing it now

**Comment by cknight** [comment]
@dsherret Thanks for the quick fix!  Can I check, do I need to wait for #21479 to be merged to main before I test this, or does #21478 provide the complete fix for this problem?
